### DISCUSSION TITLE
unable to handle command: simulator due to: 'run_status', please check syntax

### DISCUSSION THREAD

**Discussion by pygabc1** [NVIDIA/NVFlare#1373] [Q&A]
The following error occurs sometimes, and sometime the error is gone if I restart my computer. It happened frequently if I run two python (pycharm) scripts in one container. But now 5~6 times consecutively this error are occurring even I am using only ONE python (pycharm) script in a container. Any advice would be appreciated.   It is linux OS and 3.8 python, and 2.2 NVFlare version.

```
root@# cp -rf XAI_Idealist_hello-pt-v4/examples/xai-idealist WorkSpaceIdealist/
root@# nvflare simulator -w WorkSpaceIdealist/workspace -n 2 -t 2 WorkSpaceIdealist/xai-idealist
unable to handle command: simulator due to: 'run_status', please check syntax
usage: nvflare simulator [-h] [-w WORKSPACE] [-n N_CLIENTS] [-c CLIENTS] [-t THREADS] [-gpu GPU] [-m MAX_CLIENTS] job_folder

positional arguments:
job_folder

optional arguments:
-h, --help show this help message and exit
-w WORKSPACE, --workspace WORKSPACE
WORKSPACE folder
-n N_CLIENTS, --n_clients N_CLIENTS
number of clients
-c CLIENTS, --clients CLIENTS
client names list
-t THREADS, --threads THREADS
number of parallel running clients
-gpu GPU, --gpu GPU list of GPU Device Ids, comma separated
-m MAX_CLIENTS, --max_clients MAX_CLIENTS
max number of clients
```

**Comment by chesterxgchen** [comment]
Did you clone the github repo or installed from pypi ? 
if you clone from the github repo, the default branch ("dev") is unstable, development branch. Use "the latest stable "main branch 

Just trying get more information to get clear picture what's going on.

**Reply by YuanTingHsieh** [reply]
@pygabc1

**Comment by YuanTingHsieh** [comment]
please refer to discussions in #1843 if you have any problems can open again.
